src/scala/annotation/static.scala

```diff
@@ -0,0 +1,13 @@
+package scala.annotation
+
+import scala.annotation.meta._
+
+/** https://github.com/scala/scala.github.com/pull/491 */
+
+@field
+@getter
+@beanGetter
+@beanSetter
+@param
```

Comment:
Not if written by user. But I would like to leave the door open for tools\optimizers\analyzers.